x-pack/plugins/security_solution/public/attack_discovery/tour/video_toast.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButton, EuiIcon, EuiToast, EuiPortal, EuiText, EuiSpacer } from '@elastic/eui';
+import { css } from '@emotion/react';
+import React, { useCallback, useEffect } from 'react';
+import * as i18n from './translations';
+
+const VIDEO_CONTENT_HEIGHT = 160;
+const VIDEO_CONTENT_WIDTH = 250;
+const VIDEO_ID = 'BrDaDBAAvdygvemFKNAkBW';
+const VIDEO_SOURCE = `//play.vidyard.com/${VIDEO_ID}.html`;
+const VIDEO_PAGE = `https://videos.elastic.co/watch/${VIDEO_ID}`;
+
+const VideoComponent: React.FC<{ onClose: () => void }> = ({ onClose }) => {
+  const ref = React.useRef<HTMLIFrameElement>(null);
+  const [isIframeFocused, setIsIframeFocused] = React.useState(false);
+
+  const openVideoInNewTab = useCallback(() => {
+    window.open(VIDEO_PAGE, '_blank');
+  }, []);
+
+  const onIframeClick = useCallback(() => {
+    if (isIframeFocused) {
+      openVideoInNewTab();
+    }
+  }, [isIframeFocused, openVideoInNewTab]);
+
+  useEffect(() => {
+    window.addEventListener('blur', onIframeClick);
+    return () => window.removeEventListener('blur', onIframeClick);
+  }, [onIframeClick]);
+  const handleOnMouseOver = useCallback(() => {
+    setIsIframeFocused(true);
+  }, []);
+  const handleOnMouseOut = useCallback(() => {
+    setIsIframeFocused(false);
+  }, []);
+
+  return (
+    <EuiPortal>
+      <div
+        data-test-subj="attackDiscovery-tour-step-2"
+        style={{ position: 'fixed', bottom: 16, right: 16, zIndex: 9999 }}
+      >
+        <EuiToast onClose={onClose} css={{ maxWidth: VIDEO_CONTENT_WIDTH }}>
+          <div
+            css={css`
+              height: ${VIDEO_CONTENT_HEIGHT}px;
+            `}
+            onMouseOver={handleOnMouseOver}
+            onMouseOut={handleOnMouseOut}
+            onFocus={handleOnMouseOver}
+            onBlur={handleOnMouseOut}
+          >
+            <iframe
```

Comment:
Would be nice if we could set the background of the embedded page for dark mode. For the video itself it's `transparent` (and seems that way for the body/page), but it's coming through as white, so on dark mode we get these top and bottom white borders. Maybe we can change on the video site, but then would need one link for each theme (light/dark)....

Maybe we can use css/react to dig into the iframe and set the appropriate `euiThemeColor`?

https://github.com/elastic/kibana/assets/2946766/3a97a07f-5b6d-4bba-b46e-f342979463e1

Reply:
going to hold on this til i hear from design if we can use a gif here instead